shell/src/main/java/alluxio/cli/fsadmin/FileSystemAdminShellUtils.java

```diff
@@ -37,6 +47,34 @@ public static int compareTierNames(String a, String b) {
     return aValue - bValue;
   }
 
+  /**
+   * Checks if the master client service is running.
+   * Throws an exception if fails to determine that the master client service is running.
+   */
+  public static void masterClientServiceIsRunning() throws IOException {
+    try (CloseableResource<FileSystemMasterClient> client =
+             FileSystemContext.get().acquireMasterClientResource()) {
+      try {
```

Comment:
can't we consolidate this try with the try above?

Reply:
Changed to only said that the leader master is not serving requests. I think this one is enough to tell users what happens.

Reply:
@LuQQiu  the outer try can also catch exception:

```java
    try (CloseableResource<FileSystemMasterClient> client =
        FileSystemContext.get().acquireMasterClientResource()) {
      
      InetSocketAddress address = client.get().getAddress();
      System.out.println(address.getHostName());

      List<InetSocketAddress> addresses = Arrays.asList(address);
      MasterInquireClient inquireClient = new PollingMasterInquireClient(addresses, () ->
          new ExponentialBackoffRetry(50, 100, 2)
        );
      inquireClient.getPrimaryRpcAddress();
    } catch (UnavailableException e) {
        throw new IOException("The Alluxio leader master is not currently serving requests, "
            + "please check your Alluxio master status.");
    }
```